cluster-provision/k8s/1.21/manifests/prometheus/README.md

```diff
@@ -0,0 +1,28 @@
+# Prometheus
```

Comment:
Can you add documentation on how to update it?
I would expect that this is a simply copy instruction of a subfolder from the other github repo.

Reply:
Do you mean update the Prometheus Operator to a newer release? Or update the configuration?
If it is related to update to a newer release, I did a quick search and did not find any documentation about that. So, it might be done using `kubectl patch or replace`.

Reply:
I just mean, if someone goes in and wants to replace the manifests here with the latest one from `kube-prometheus`. Did you do some extra modifications, or simply copy-paste them?

Reply:
I see, the only change that I did was to decrease the number of replicas from 2 to 1.
I will write that in the README